components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -78,8 +79,23 @@ void AdsServiceFactory::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kBraveAdsEnabled, false);
   registry->RegisterUint64Pref(prefs::kBraveAdsPerHour, 2);
-  registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, 6);
+
+  #if defined(OS_ANDROID)
+    registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, 12);
+  #else
+    registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, 20);
+  #endif
+
   registry->RegisterIntegerPref(prefs::kBraveAdsIdleThreshold, 15);
+
+  if (prefs::kBraveAdsPrefsCurrentVersion == 2) {
```

Comment:
shouldn't you be checking the actual pref value? This will always be true because `prefs::kBraveAdsPrefsCurrentVersion` is a const

Reply:
👍 improved logic so that if the pref does not exist defaults to version 1, where this value is updated after each successful migration